packages/launch/src/launch/index.tsx

```diff
@@ -21,9 +21,10 @@ import './styles.scss';
 interface Props {
 	locale?: string;
 	siteId: number;
-	openCheckout: ( siteSlug: string, isEcommerce?: boolean ) => void;
+	openCheckout: ( siteSlug?: string, isEcommerce?: boolean ) => void;
```

Comment:
It's because we use the subdomain as the value for site slug and that's not guaranteed to have a value because it's coming from API. We had a fallback to `siteId`: https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/hooks/use-cart.ts#L51

But I've moved that now inside `openCheckout` function to keep it's signature cleaner.

Reply:
Thanks for elaborating!